tools/install-driver.mjs

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env node
+/* eslint-disable no-console */
+
+/**
+ * Copyright 2023 Google LLC.
+ * Copyright (c) Microsoft Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Installs a driver matching browser version defined in
+ * `.browser` using `@puppeteer/browsers` to the directory provided as the first
+ * argument (default: cwd). The executable path is written to the
+ * `executablePath` output param for GitHub actions.
+ *
+ * Examples:
+ *  - `node tools/install-driver.mjs`
+ *  - `node tools/install-driver.mjs /tmp/cache`
+ */
+
+import {readFile} from 'fs/promises';
+import {homedir} from 'os';
+import {resolve} from 'path';
+
+import {setOutput, setFailed} from '@actions/core';
+import {install, computeExecutablePath} from '@puppeteer/browsers';
+
+const SHELL_ARG = '--shell';
+
+try {
+  const browserSpec = (await readFile('.browser', 'utf-8')).trim();
```

Comment:
We read `.browser` but we hard-code it to `chromedriver`. This is confusing.

Please either:

- create a new file to hold `chromedriver@<version>`, OR
- document this quirk here

Documenting it should be enough. But ensure to remove `See .browser for the format.` below, as it is also confusing in this context.

Reply:
Moved all the logic to `install-browser.mjs` with option `--chrome-driver`.